src/Repository/ConferenceRepository.php

```diff
@@ -101,6 +101,21 @@ public function getNullCoordinatesConferences(): ?\Generator
         }
     }
 
+    public function findExistingConference(Conference $conference): ?Conference
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.startAt = :findStartAt AND c.endAt = :findEndAt')
+            ->andWhere('levenshtein(c.name, :findName) < 4')
+            ->setParameters([
+                'findStartAt' => $conference->getStartAt(),
+                'findEndAt' => $conference->getEndAt(),
+                'findName' => $conference->getName(),
+            ])
+            ->getQuery()
```

Comment:
This is almost impossible (I guess) but let's be defensive: `setMaxResults(1)`, otherwise you'll get an error when the query returns more than 1 result because of `getOneOrNullResult`